packages/graphql-server/src/resolvers/negotiators/__mocks__/create-arg.ts

```diff
@@ -0,0 +1,23 @@
+export const createArgStub = {
```

Comment:
Can you add a type definition to these stubs please so they are self documenting and we get compile time feedback if the contract changes